File: FamilyTools.EasyCompta/Model/BaseModel.cs
namespace EasyCompta.Server.Model
{
    public class BaseModel
    {
        public int Id { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime UpdateDate { get; set; }

        public ValueTask DisposeAsync()
        {
            throw new NotImplementedException();
        }
    }
}

File: FamilyTools.Web/src/app/models/base-model.ts
export class BaseModel {
  id?: number;
  creationDate?: Date;
  updateDate?: Date;

  constructor(id?: number, creationDate?: Date, updateDate?: Date){
    this.id = id;
    this.creationDate = creationDate;
    this.updateDate = updateDate;
  }

}